src/libs/intents/localMethods.ts

```diff
@@ -99,7 +100,7 @@ const isAvailable = (featureName: string): Promise<boolean> => {
   } else if (featureName === 'shareFiles') {
     return Promise.resolve(true)
   } else if (featureName === 'iap') {
-    return Promise.resolve(true)
+    return isIapAvailable()
```

Comment:
not in the scope of this feature, but I think we should start thinking about a more elegant way to handle the isAvailable method implementation (more declarative at least)